cypress/integration/Stepper._spec.js

```diff
@@ -0,0 +1,227 @@
+                cy.get('.v-checkbox:nth-child(2) > input').check()
+                cy.get('.next-button').click()
+                cy.get('.license-use-card').should('be.visible').contains(' CC0 1.0 ')
+                cy.get('.previous-button').click()
+                cy.get('.previous-button').click()
+                cy.get('.previous-button').click()
+                cy.get('.step-0')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-1')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-2')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-3')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-4')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.recommended-card').should('be.visible').contains('CC BY-SA 4.0')
+                cy.get('.license-use-card').should('be.visible').contains('CC BY-SA 4.0')
+            })
+        })
+
+        describe('Checks whether the user can click on Start Again Button and application gets a hot reload', () => {
+            it('Clicks on Start Again Buttton', () => {
+                cy.get('.restart-button').click()
+                cy.get('.step-1').should('have.class','inactive')
+                cy.get('.step-2').should('have.class','inactive')
+                cy.get('.step-3').should('have.class','inactive')
+                cy.get('.step-4').should('have.class','inactive')
+            })
+        })
+
+        describe('Checks that the application gets an auto-scroll to "Mark Your Work Section" when clicked on Done', () => {
+            it('Choose a License of your choice ', () => {
+                cy.get('.step-0')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-1')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-2')
+                .find('[value="yes"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-3')
+                .find('[value="no"]')
+                .check()
+                cy.get('.next-button').click()
+                cy.get('.step-4').should('have.class','disabled')
+                cy.get('.recommended-card').should('be.visible').contains('CC BY-ND 4.0')
+                cy.get('.license-use-card').should('be.visible').contains('CC BY-ND 4.0')
+                cy.get('.next-button').click()
+                cy.get('.license-use-card').scrollIntoView()
```

Comment:
You do not need to scroll an element into view. In fact, when you click the 'Done' button, it should scroll automatically, and we need to check if it does. 
Actually, it is better to test it on a mobile viewport. Could you set up this test with the mobile viewport?

Reply:
@obulat You want me to convert viewport or this particular test or for the complete test file.

Reply:
Yes, I would like this particular test to have a different viewport.

Reply:
Shall I give iphone 10 viewport. Will that be okay or any other .

Reply:
Any viewport width of under 768px will do, as that's the breakpoint when the two-column layout becomes a one-column layout.